clearwater-snmpd/etc/snmp/snmpd.conf.clearwater-snmpd

```diff
@@ -259,3 +260,7 @@ dlmod chronos_handler /usr/lib/clearwater/chronos_handler.so
 # Don't log on successful connections - allowing this log means that a log is
 # made to syslog on every stats query, which is too spammy
 dontLogTCPWrappersConnects 1
+
+# File descriptor MIB handling
+pass .1.2.826.0.1.1578918.17.1 /bin/bash /usr/share/clearwater/infrastructure/scripts/snmpd_fd_handler
```

Comment:
Can we just make snmpd_fd_handler executable (`chmod a+x`) and call it directly?